trailblazer/server/utils.py

```diff
@@ -25,3 +33,26 @@ def stringify_timestamps(data: dict) -> dict[str, str]:
         if isinstance(val, datetime.datetime):
             data[key] = str(val)
     return data
+
+
+def handle_endpoint_errors(func):
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except AuthenticationError:
+            return jsonify("User not allowed"), HTTPStatus.FORBIDDEN
+        except MissingAnalysis as error:
+            return jsonify(error=str(error)), HTTPStatus.NOT_FOUND
+        except ValidationError as error:
+            LOG.error(f"Validation error in analysis endpoint: {error}")
+            return jsonify(error=str(error)), HTTPStatus.BAD_REQUEST
+        except Exception as error:
+            LOG.error(f"Unexpected error in analysis endpoint: {error}")
+            return (
+                jsonify(error="An error occurred while processing your request."),
+                HTTPStatus.INTERNAL_SERVER_ERROR,
```

Comment:
Do we have any logic depending on the Status being CONFLICT here?

Reply:
I went through the cg client and the cigrid web app and could not find any such logic depending on the status code being 409